public/main.css

```diff
@@ -33,9 +35,13 @@ header {
   padding: 20px;
 }
 header .content {
+  display: -webkit-flex;
+  display: -ms-flexbox;
   display: flex;
 }
 header .spacer {
+  -webkit-flex: 1;
+  -ms-flex: 1;
   flex: 1;
 }
 header a {
```

Comment:
I'd prefer triggering overall rebuilds rather than people adding the bits and parts in PR's, personally.